example/README.md

```diff
@@ -0,0 +1,48 @@
+## WARNING: This is Alpha software and not intended for use until a stable release.
+
+# Running M3DB on GCP
+
+Setup GCP for m3db:
+
+    1. Download necessary packages
+        $ sudo apt-get install golang golint make git golang-glide
+    2. Configure git
```

Comment:
Do they need to do this?

Reply:
No- just if they want to make git changes. I'll remove.